### PR TITLE
Blog post about a minimalistic ruby compiler

### PR DESCRIPTION
**Pull request by AlexDenisov**
We've built a minimalistic AOT compiler for Ruby (https://github.com/DragonRuby/lightstorm), which is using LLVM/MLIR, and would love to publish this article on LLVM's blog.

Please, let me know if there is anything else I need to do to make it mergeable 🙌

### CODE REVIEW

File: content/posts/2024-12-03-minimalistic-ruby-compiler.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
+---
+author: "Alex Denisov, Amir Rajan"
+date: "2024-12-03"
+tags: ["llvm", "mlir", "ruby", "dragonruby"]
+title: "Lightstorm: minimalistic Ruby compiler"
+---
+
+Some time ago I was talking about an [ahead-of-time Ruby compiler](https://www.youtube.com/watch?v=NfMX-dFMSr0).
```
Comment (asl) [inline content/posts/2024-12-03-minimalistic-ruby-compiler.md:8]: I would probably suggest to outline more LLVM / MLIR in the beginning

**Review by asl — COMMENTED** [review]
Thanks for the post! Given that this is an LLVM project blog, it would be great to emphasize LLVM / MLIR role better. As far as I can see, the very first occurrence of "MLIR" is in Approach. And looks like this is the only mention of the project in the whole post ;)

**Comment by AlexDenisov** [comment]
Totally makes sense, will do! 👍

**Comment by AlexDenisov** [comment]
I added a bit more details about the role of MLIR.

I'm trying to keep the blog post concise, though I definitely see the value in getting deeper into the technical details for the audience of this blog.

**Comment by AlexDenisov** [comment]
Hi @asl, does my latest change make it more appropriate for this blog?

File: content/posts/2024-12-03-minimalistic-ruby-compiler.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
+---
+author: "Alex Denisov, Amir Rajan"
+date: "2024-12-03"
```
Comment (asl) [inline content/posts/2024-12-03-minimalistic-ruby-compiler.md:3]:
```suggestion
date: "2024-12-09"
```

**Review by asl — APPROVED** [review]

**Comment by AlexDenisov** [comment]
Thank you @asl, do I just merge it now to make it published or is there anything else needed?

**Pull request merged**